src/PlatformInstaller/DotNetUpgrade/Views/DotNetPreReqView.xaml

```diff
@@ -0,0 +1,26 @@
+﻿<UserControl
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:global="clr-namespace:"
+        x:Class="DotNetPreReqView">
+    <UserControl.DataContext>
+        <global:DotNetPreReqViewModel/>
+    </UserControl.DataContext>
+    <UserControl.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="../../Styles.xaml"/>
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </UserControl.Resources>
+    <DockPanel >
+        <TextBlock DockPanel.Dock="Top" FontSize="18" FontWeight="Bold">Install .NET Framework 4.5.2</TextBlock>
+        <StackPanel DockPanel.Dock="Bottom" Orientation="Horizontal" FlowDirection="RightToLeft">
+            <Button x:Name="Exit" Content="Exit"/>
+            <Button x:Name="Install" Content="Install"/>
+        </StackPanel>
+        <StackPanel>
+            <TextBlock Foreground="Gray">Please click Install to commence the download and installation of this prerequisite. </TextBlock>
```

Comment:
Not sure.   We provide the link on failure to download or install. I thought that would be enough